manifests/04-rbac-rolebinding-cluster.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Moved from https://github.com/openshift/console-operator/pull/270/files#diff-917b39b291003c5a9d8a57afceaa65aaL28 to keep these tidy.